qt5 fails to build

Here are the [logs](http://sprunge.us/XlFTu7)

You should really mention the maintainer of the package so that they can be notified.... @dilyn-corner 

Here's the actual error:
```
File '**' is not valid
make[2]: Entering directory '/home/finn/.cache/kiss/build-8715/qt5/src/widgets'
perl /home/finn/.cache/kiss/build-8715/qt5/mkspecs/features/data/unix/findclasslist.pl < QtWidgets.version.in > QtWidgets.version
g++ -pipe -O2 -std=c++1z -fvisibility=hidden -fvisibility-inlines-hidden -fno-exceptions -Wall -Wextra -Wvla -Wdate-time -Wshift-overflow=2 -Wduplicated-cond -Wno-stringop-overflow -Wno-format-overflow -dM -E -o .moc/moc_predefs.h ../../mkspecs/features/data/dummy.cpp
/home/finn/.cache/kiss/build-8715/qt5/src/widgets/uic_wrapper.sh dialogs/qfiledialog.ui -o .uic/ui_qfiledialog.h
File '**********************' is not valid
make[2]: *** [Makefile:78299: .uic/ui_qfiledialog.h] Error 1
make[2]: Leaving directory '/home/finn/.cache/kiss/build-8715/qt5/src/widgets'
make[1]: *** [Makefile:530: sub-widgets-make_first] Error 2
make[1]: *** Waiting for unfinished jobs....```

--- a/community/qt5/depends
+++ b/community/qt5/depends
@@ -2,6 +2,7 @@ bison make
 flex  make
 freetype-harfbuzz
 gperf make
+icu
 libICE
 libSM
 libX11

